source/operations.rst

```diff
@@ -0,0 +1,354 @@
+   hub-config      4     6m47s
+   user-scheduler  1     3m35s
+
+   ==> v1/PersistentVolumeClaim
+   NAME        STATUS   VOLUME               CAPACITY  ACCESS MODES  STORAGECLASS  AGE
+   hub-db-dir  Pending  managed-nfs-storage  6m46s
+
+   ==> v1/ClusterRole
+   NAME                              AGE
+   hub-user-scheduler-complementary  3m34s
+
+   ==> v1/ClusterRoleBinding
+   NAME                              AGE
+   hub-user-scheduler-base           3m34s
+   hub-user-scheduler-complementary  3m34s
+
+   ==> v1/RoleBinding
+   NAME  AGE
+   hub   6m46s
+
+   ==> v1/Pod(related)
+   NAME                            READY  STATUS   RESTARTS  AGE
+   continuous-image-puller-flf5t   1/1    Running  0         3m34s
+   continuous-image-puller-scr49   1/1    Running  0         3m34s
+   hub-569596fc54-vjbms            0/1    Pending  0         3m30s
+   proxy-79fb6d57c5-nj8n2          1/1    Running  0         2m22s
+   user-scheduler-9685d654b-9zt5d  1/1    Running  0         3m30s
+   user-scheduler-9685d654b-k8v9p  1/1    Running  0         3m30s
+
+   ==> v1/Secret
+   NAME        TYPE    DATA  AGE
+   hub-secret  Opaque  3     6m47s
+
+   ==> v1/DaemonSet
+   NAME                     DESIRED  CURRENT  READY  UP-TO-DATE  AVAILABLE  NODE SELECTOR  AGE
+   continuous-image-puller  2        2        2      2           2          <none>         3m34s
+
+   ==> v1/Deployment
+   NAME            DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
+   hub             1        1        1           0          6m45s
+   proxy           1        1        1           1          6m45s
+   user-scheduler  2        2        2           2          3m32s
+
+   ==> v1/StatefulSet
+   NAME              DESIRED  CURRENT  AGE
+   user-placeholder  0        0        6m44s
+
+   ==> v1beta1/Ingress
+   NAME        HOSTS                                 ADDRESS  PORTS  AGE
+   jupyterhub  notebooktest.fedcloud-tf.fedcloud.eu  80, 443  6m44s
+
+   ==> v1beta1/PodDisruptionBudget
+   NAME              MIN AVAILABLE  MAX UNAVAILABLE  ALLOWED DISRUPTIONS  AGE
+   hub               1              N/A              0                    6m48s
+   proxy             1              N/A              0                    6m48s
+   user-placeholder  0              N/A              0                    6m48s
+   user-scheduler    1              N/A              1                    6m47s
+
+   ==> v1/Role
+   NAME  AGE
+   hub   6m46s
+
+
+   NOTES:
+   Thank you for installing JupyterHub!
+
+   Your release is named hub and installed into the namespace hub.
+
+   You can find if the hub and proxy is ready by doing:
+
+   kubectl --namespace=hub get pod
+
+   and watching for both those pods to be in status 'Running'.
+
+   You can find the public IP of the JupyterHub by doing:
+
+   kubectl --namespace=hub get svc proxy-public
+
+   It might take a few minutes for it to appear!
+
+   Note that this is still an alpha release! If you have questions, feel free to
```

Comment:
Aren't we at Beta now? :)

Reply:
That's jupyterhub helm notes, we are production :)

Reply:
OK cool :)